.travis.yml

```diff
@@ -10,14 +10,21 @@ os:
   - osx
 
 script: ./bin/travis.sh
-env: TASK=test
+env: TASK=test COVERAGE=false
 install: npm install --ignore-scripts
 
 after_success:
-  - npm run coverage:ci
+  - if [ "${COVERAGE}" == "true" ]; then npm run coverage:ci; fi
```

Comment:
I find the need to exclude Node.js 10.x build and include it again as too verbose to my taste.

Wouldn't it be easier to not introduce `COVERAGE` variable and rewrote the condition to check `$TRAVIS_NODE_VERSION` and `$TRAVIS_OS_NAME` instead?

Not a big deal though.